debezium-server/debezium-server-nats_streaming/pom.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please use dash instead of underscore in directory name?

Reply:
Done!

Reply:
Done!